### ISSUE TITLE
Space game won't proceed after hitting done after first turn 0.45.2

### ISSUE THREAD

**Issue by BLR-IIC** [MegaMek/megamek#1210]
I started a space game against Princess.  My force had two _McKenna_ battleships and Princess had two _Texas_ battleships and some kind of destroyer.  Each of my _McKennas_ had six Santa Anna missiles in their AR10 bays, but those were never fired.

We played through the first turn with no damage done.  During the end phase, I clicked on the "Done" button and the game would not move further.  Logs and savegame attached.

[MegaMek Logs 12.22.2018.zip](https://github.com/MegaMek/megamek/files/2705656/MegaMek.Logs.12.22.2018.zip)

**Comment by HammerGS** [comment]
Look like a related to this https://github.com/MegaMek/megamek/issues/1169

```
Exception in thread "Packet Pump" java.lang.ArrayIndexOutOfBoundsException: 8
	at megamek.common.Warship.getASEWAffected(Warship.java:78)
	at megamek.server.Server.decrementASEWTurns(Server.java:1906)
	at megamek.server.Server.endCurrentPhase(Server.java:3263)
	at megamek.server.Server.changePhase(Server.java:2486)
	at megamek.server.Server.endCurrentPhase(Server.java:3164)
	at megamek.server.Server.changePhase(Server.java:2486)
	at megamek.server.Server.endCurrentPhase(Server.java:3143)
	at megamek.server.Server.checkReady(Server.java:2256)
	at megamek.server.Server.handle(Server.java:32319)
	at megamek.server.Server$PacketPump.run(Server.java:368)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by NickAragua** [comment]
This was actually fixed with PR #1196, so it should be remedied with the next dev release.
